working-groups/best-practices.md

```diff
@@ -0,0 +1,24 @@
+# Working Group best practices
+
+This document describes best practices for the operation of AMP Working Groups.  Working Groups may deviate from these best practices in reasonable ways that work best for the Working Group as long as those changes align with [AMP’s governance policy](../GOVERNANCE.md).
+
+* GitHub repository.  Create an ampproject/wg-{name} GitHub repository (e.g. “ampproject/wg-ui”).
+  * Link to this repo from the [ampproject/meta/working-groups/README.md](README.md) file.
+  * Use this repo to keep for documentation about the working group, meeting agendas, etc.  **This repo should not contain code.**
+  * This repository should contain a README.md with enough information to make it possible to keep track of all work that is happening within the WG and all of the decisions that require the attention of WG members.  This should include:
+    * the WG’s mandate from the TSC (e.g. which repositories and parts of the codebase the WG is responsible for)
+    * a list of members
+    * how the working group operates, including discussion channels, how decisions are made, and how issue labels/GitHub projects are used.  This documentation should.
```

Comment:
typo: extra `This documentation should.` at the end of the sentence.

Reply:
Done!